parse/parse.go

```diff
@@ -401,3 +402,16 @@ func fatal(msg string, err error) {
 	log.Fatal(msg, err)
 	os.Exit(1)
 }
+
+// Truncate lines to prevent sending over massive hunks, e.g. a minified file.
+func truncateLine(line string) string {
+	// len(line) returns number of bytes, not num. characters, but it's a close enough
+	// approximation for our purposes
+	if len(line) > maxLineCharCount {
+		// convert to rune slice so that we don't truncate multibyte unicode characters
+		runes := []rune(line)
+		return string(runes[0:maxLineCharCount])
```

Comment:
Should we put some character at the end here to indicate that the line has been terminated?

Reply:
done in 972b264